app/src/main/java/org/mozilla/fenix/customtabs/FennecWebAppIntentProcessor.kt

```diff
@@ -52,16 +53,17 @@ class FennecWebAppIntentProcessor(
      * A custom tab config is also set so a custom tab toolbar can be shown when the user leaves
      * the scope defined in the manifest.
      */
-    override suspend fun process(intent: Intent): Boolean {
+    override fun process(intent: Intent): Boolean {
         val safeIntent = intent.toSafeIntent()
         val url = safeIntent.dataString
 
         return if (!url.isNullOrEmpty() && matches(intent)) {
-            val webAppManifest = loadManifest(safeIntent, url)
+            val webAppManifest = runBlocking { loadManifest(safeIntent, url) }
```

Comment:
@NotWoods did we want an early return here similar to the a-c PR? 

https://github.com/mozilla-mobile/android-components/pull/7498/files#diff-b63d65380bcd2a6c53e6212b40771970R47

Reply:
I think its OK to be a bit more lax here in case of migration bugs